src/sage/rings/lazy_series.py

```diff
@@ -5390,6 +5390,77 @@ def derivative(self, *args):
                                                    P.is_sparse())
         return P.element_class(P, coeff_stream)
 
+    def adams_operator(self, p):
+        """
+        Return the image of ``self`` under the Adams operator of index ``p``.
+
+        This raises all variables to the power ``p``, both the power
+        series variables and the variables inside the coefficient ring.
+
+        INPUT:
+
+        - ``p`` -- a positive integer
+
+        EXAMPLES:
+
+        With no variables in the base ring::
+
+            sage: A = LazyPowerSeriesRing(QQ,'t')
+            sage: f = A([1,2,3,4])
+            sage: f.adams_operator(2)
+            1 + 2*t^2 + 3*t^4 + 4*t^6
+
+        With variables in the base ring::
+
+            sage: q = polygen(QQ,'q')
+            sage: A = LazyPowerSeriesRing(q.parent(),'t')
+            sage: f = A([0,1+q,2,3+q**2])
+            sage: f.adams_operator(2)
+            ((q^2+1)*t^2) + 2*t^4 + ((q^4+3)*t^6)
+
+        In the multivariate case::
+
+            sage: A = LazyPowerSeriesRing(ZZ,'t,u')
+            sage: f = A({(1,2):4,(2,3):6})
```

Comment:
Maybe this can be written more explicitly:
```
sage: A.<t, u> = LazyPowerSeriesRing(ZZ)
sage: f = 4*t*u^2 + 6*t^2*u^3
```

Reply:
Could you also give an example explaining how to make a symmetric function act on a lazy power series?

Reply:
well, this is the next stage, for another pull request.. The main idea is that power sum `p_n` acts by Adams operator.